Sources/Authentication/Views/AutomatedLoginButton.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Nicolai Dam on 03/06/2023.
+//
+
+import SwiftUI
+
+/// QA login button view that uses the AutomatedLoginSelection view under the hood
+/// The callback is triggered whenever either manual or automated login is chosen and should be used to override the AuthenticationHandler environment in the app
+/// - Example of AutomatedLoginButton used as a ToolbarItem:
+/// ````
+///#if !RELEASE
+///.toolbar  {
+///    ToolbarItem(placement: .bottomBar) {
+///        AutomatedLoginButton(
+///            jsonFileURL: Bundle.main.url(forResource: "automated_login_ejerskifte", withExtension: "json")!
+///        ) { callback in
+///            switch callback {
+///            case .Manual:
+///                // Manual login chosen
+///                // TODO: Override AuthenticationHandler instance for app
+///            case .Automated(let userItem, let url):
+///                // Automated user is chosen
+///                // TODO: Override AuthenticationHandler instance for app
+///            }
+///        }
+///    }
+///}
+///#endif
+/// ````
+public struct AutomatedLoginButton: View {
```

Comment:
Instead of presenting the AutomatedLoginSelection sheet in each app you can also use this helper view with less code